addons/bus/bus.py

```diff
@@ -114,7 +114,7 @@ def loop(self):
         with openerp.sql_db.db_connect('postgres').cursor() as cr:
             conn = cr._cnx
             cr.execute("listen imbus")
-            cr.commit();
+            cr.commit()
```

Comment:
Uh sorry, PyCharm was complain on semicolon, and that change was too small for separate commit, but I will not do like that again, sorry!